[RM Ch3] Update SFC Section 3.5.1 Architecture (#2319)
cntt-n/CNTT: doc/ref_model/chapters/chapter03.md

However, in a service-specific overlay network, packets are routed based on policies. This requires specific support at network level such as  at CNI in CNF environment to provide such specific routing mechanism.

<a name="3.5.1.2"></a>
#### 3.5.1.2 SFC Architecture

>  The SFC Architecture is built out of architectural building blocks that are logical components to relaize the service function chaining. These consist of management, control and data components. 

Reviewer: Suggestion:
```suggestion
 The SFC Architecture is composed of functional management, control and data components as categorised in the Table 3-6 below. 
```

The table below highlights areas under which common SFC functional components can be categorized.

> | Components | Example         | Resposbilities |

Reviewer 2: Typo: Responsabilities

Author: accepted and corrected.

> The table below highlights areas under which common SFC functional components can be categorized.

Reviewer: delete if previous suggestion accepted

|:---:|:----:|:---|

> |**Manangement** | `SFC orchestrator`  | High Level of orchestrator <br /> Orchestrate the SFC based on SFC Models/Policies with help of control components.| 

Reviewer 2: Typo: Management

Author: accepted and corrected.

| | `SFC OAM Components` | Responsible for SFC OAM functions |
|| `VNF MANO` | NFVO, VNFM, and VIM <br />Responsible for SFC Data components lifecycle |

> || `CNF MANO` | CNF DevOps Components <br />Responsible for SFC data componets lifecycle |

Reviewer 2: Typo: components

Author: accepted and corrected.

> | **Control** | `SFC SDN Controller` | SDNC resopsosible to create the service specific overlay network. <br /> Deploy different techniques to stitch the wiring but provide the same functionality, for example l2xconn, srv6 , Segment routing etc.  |

Reviewer 2: Typo: responsible, SRv6

Author: accepted and corrected.

|| `SFC Renderer` | Creates and wires ports/interfaces for SF data path |

> | **Data** | `Core components`<br /> SF, SFF, SF Proxy  | Ressponsible for steering the traffic for intended service functionalities based on Policies |

Reviewer 2: Typo: Responsible

Author: accepted and corrected.

<p align="center"><b>Table 3-6:</b> SFC Architecture Components</p>

> **Note:** These are logical components and listed for their functionalies only.  

The SFC Architecture components can be viewed as:- 
<p align="center"><img src="../figures/ch03-model-sfc-architecture-vnf-2.png" alt="SFC Architecture for VNF based SFs" Title="SFC Architecture for VNF based SFs" width="45%"/>

> <img src="../figures/ch03-model-sfc-architecture-cnf-2.png" alt="SFC Architecture for CNF based SFs" Title="SFC Architecture for CNF based SFs" width="45%"/></p>

Reviewer: add a new line between two figures. It would be better is a little bit of description were added for each of the two figures (the text will also provide a break between the two figures)

Author: @pgoyal01 will update accordingly.

<p align ="center"><b>Figure 3-6:</b> SFC Architecture for VNF based SFs <b>Figure 3-7:</b> SFC Architecture for CNF based SFs</p>

> SFC management components together with control components are responsible for rendering SFC requests to Service function paths, converting requisite SFC policies into network topology dependent paths and forwarding steering policies to relevant SFC data components - classifiers, service function forwarders. SFC data plane components are responsible for carrying out the steering policies.

Reviewer: also need some rewrite of 

```suggestion
The SFC management components together with the control components are responsible for rendering SFC requests to Service Function paths. For this they convert requisite SFC policies into network topology dependent paths and forwarding steering policies. Relevant SFC data components - classifiers, service function forwarders - are responsible for managing the steering policies.
```
